znft/config.go

```diff
@@ -42,10 +42,18 @@ type Configuration struct {
 	Value                            int64  // Value to execute Ethereum smart contracts (default = 0)
 }
 
+type App struct {
```

Comment:
Since we are on sdk, we'd better not use the 'App', as it is more like the application level name. I would suggest to name it as `Znft`.  Also rename the `config.go` to `znft.go`.

Reply:
Done